docs/general/networking/vpn.md

```diff
@@ -0,0 +1,52 @@
+---
+uid: network-vpn
+title: VPNs
+---
+
+# VPNs
+
+"A virtual private network (VPN) is a mechanism for creating a secure connection between a computing device and a computer network, or between two networks, using an insecure communication medium such as the public Internet." - [Wikipedia](https://en.wikipedia.org/wiki/Virtual_private_network)
+
+## Commercial VPN services
+
+Commercial VPN services such as Mullvad, PIA, NordVPN or Windscribe are a great way to hide your IP and bypass georestrictions. While some of them do provide a port forwarding feature, hosting Jellyfin using one is not recommended.
+
+## Point To Point VPNs
+
+Services like Tailscale, Zerotier and Hamachi provide an easy eay to link devices together in a Virtual network, providing LAN access between them. They are an easy way to allow access to devices without requiring much networking expertise. However, they require devices that support their apps in order to function.
+
+### Tailscale
+
+[Tailscale](https://tailscale.com/) is a VPN solution based on [WireGuard](https://www.wireguard.com/). The free tier of its service allows up to 100 connected devices. Tailscale is easy to setup but support is limited.
+
+To setup remote access to Jellyfin using Tailscale, please follow the steps below.
+
+1. Register an account at [their website](https://tailscale.com/).
+2. Download and install the Tailscale client from [here](https://tailscale.com/download/) on the Jellyfin server and all clients.
+3. Login to your Tailscale account on all devices. They will be automatically assigned an IP address for the Tailscale network.
+4. Connect to the IP assigned to the Jellyfin server from all clients.
+
+### ZeroTier
+
+[ZeroTier](https://www.zerotier.com/) is another VPN solution. The free tier allows up to 25 connected devices. ZeroTier supports much more platforms than Tailscale.
+
+To setup remote access to Jellyfin using ZeroTier, please follow the steps below.
+
+#### Setting Up a Network
+
+1. Register an account at [their website](https://www.zerotier.com/).
+2. Login to [ZeroTier Central](https://my.zerotier.com/). For new accounts, there should be 4 networks already created. You can use any of them, or create a new one and enter its settings page by clicking on it.
+3. Give the network a name and description that you can remember (optional).
+4. Change `Access Control` to `Private`
+5. Scroll down and check `IPv4 Auto Assign`. Select `Easy` and pick an IP range from below. Any IP range should work.
```

Comment:
Hint: The range must not be in use by any of the connected clients nor the server!